include/ur_client_library/ur/ur_driver.h

```diff
@@ -180,6 +181,29 @@ class UrDriver
    */
   bool writeJointCommand(const vector6d_t& values, const comm::ControlMode control_mode);
 
+  /*!
+   * \brief Writes a trajectory point onto the dedicated socket.
+   *
+   * \param values Desired joint or cartesian positions
+   * \param goal_time Time for the robot to reach this point
+   * \param cartesian True, if the point sent is cartesian, false if joint-based
+   *
+   * \returns True on successful write.
+   */
+  bool writeTrajectoryPoint(const vector6d_t& values, const bool cartesian, const float goal_time = 0.0,
+                            const float blend_radius = 0.052);
+
+  /*!
+   * \brief Writes a control message in trajectory forward mode.
+   *
+   * \param trajectory_action The action to be taken, such as starting a new trajectory.
+   * \param point_number The number of points of a new trajectory to be sent.
```

Comment:
```suggestion
   * \param trajectory_action The action to be taken, such as starting a new trajectory
   * \param point_number The number of points of a new trajectory to be sent
```

Consistency with other param documentations